### PR TITLE
Reset all sessions (in occ command and on upgrade)

### PR DESCRIPTION
**Pull request by mejo-**
### 📝 Summary

* feat(reset): Add `--all` option to `text:reset` occ command
* fix(reset): Reset all document sessions on upgrades from 4.0.0 or below
* Fixes: #5420
* Fixes: nextcloud/collectives#1270

:warning: Reminder: Versions and version comparison need adjustments when backporting to stable branches.

### 🏁 Checklist

- [x] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [x] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits

### CODE REVIEW

File: lib/Command/ResetDocument.php [hunk 1]
```diff
@@ -56,30 +63,44 @@ protected function configure(): void {
...
-		try {
-			$this->documentService->resetDocument($fileId);
-		} catch (DocumentHasUnsavedChangesException) {
-			$output->writeln('Not resetting due to unsaved changes');
-			return 1;
+		$rc = 0;
+		foreach ($fileIds as $id) {
+			if ($fullReset) {
+				$output->writeln('Force-reset the document session for file ' . $id);
+				$this->documentService->resetDocument($id, true);
```
Comment (juliushaertl) [inline lib/Command/ResetDocument.php:92]: I guess we can skip the rest with continue?

File: lib/Command/ResetDocument.php [hunk 2]
```diff
@@ -56,30 +63,44 @@ protected function configure(): void {
...
+			}
+
+			$output->writeln('Reset the document session for file ' . $id);
+			try {
+				// $this->documentService->resetDocument($id);
```
Comment (juliushaertl) [inline lib/Command/ResetDocument.php:97]: Commented out on purpose?

Comment (mejo-) [reply lib/Command/ResetDocument.php:97]: Ouch, debugging leftover.

File: lib/Command/ResetDocument.php (hunk 1 above)
Comment (mejo-) [reply lib/Command/ResetDocument.php:92]: Indeed, good catch :fishing_pole_and_fish:

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**